security/security-design/shared-assets/email-security-dns-check/README.md

```diff
@@ -0,0 +1,72 @@
+## Purpose of the email security DNS check
+
+Oracle requires that certain records are added to a customers DNS to allow Oracle's email delivery service to send emails on the customers behalf. This prevents the customers emails from being flagged as spoofed. 
+
+### Disclaimer
```

Comment:
Not bad. I miss that this service is "Free for non-commercial use."

Reply:
Added the remark about free for non commercial usage.